assets/tests/unit/home.spec.js

```diff
@@ -1,12 +1,34 @@
 import { shallowMount } from '@vue/test-utils';
-import AchievedGoals from '@/components/AchievedGoals.vue';
+import VueRouter from 'vue-router';
 import BucketList from '@/components/BucketList.vue';
+import HEADERS from '@/constants/http.js';
 jest.mock('axios');
 import axios from 'axios';
 
 beforeEach(() => {
   jest.resetModules();
   jest.clearAllMocks();
+  let promise = new Promise(function(resolve, reject) {
```

Comment:
Might be useful to have some sort of `PromiseFactory` helper class (or a functional equivalent, depending on your preferences).